### PR TITLE
Orders table

### PR DESCRIPTION
**Pull request by enaviznev-bmt**
added lambda functions for working with orders and added new table

### CODE REVIEW

File: api/admin/admin.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      });
+    }
+
+     return Promise.all([this.db.scan(paramsUsersTable).promise(), this.db.scan(paramsOrdersTable).promise()])
+      .then(([profiles, orders]) => {
+        console.log('data from usersTable = ', profiles);
+        console.log('data from ordersTable = ', orders);
+        orders.Items.forEach((order) => {
+          let orderedByProfile = profiles.Items.find((profile) => profile.id === order.orderedBy);
+          console.log(orderedByProfile.firstName);
```
Comment (zaikin-andrew) [inline api/admin/admin.ts]: If it's a debug console add a description. if no remove it

File: api/admin/admin.ts [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+          order.formProfile.firstName = orderedByProfile.firstName;
+          order.formProfile.lastName = orderedByProfile.lastName;
+          formedOrders.push(order);
+          console.log('asd', formedOrders);
```
Comment (zaikin-andrew) [inline api/admin/admin.ts]: If it's a debug console add a description. if no remove it

File: api/admin/admin.ts [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+      ExpressionAttributeValues: {
+        ":id": id.toString()
+      }
+    };
+
+    return this.db.scan(params).promise();
+  }
+}
+
+function getRandom (max) {
```
Comment (zaikin-andrew) [inline api/admin/admin.ts]: Probably would be better to include these functions in the class as static methods or move it helper file with export.

File: api/admin/admin.ts [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+      id: faker.random.number(),
+      type: getKeyType(),
+      name: faker.lorem.words(),
+      cover: faker.image.imageUrl(),
+      description: faker.lorem.paragraph(),
+      price: getRandom(100)
+    })
+  }
+  return orders;
+}
```
Comment (azhivaeva-bmt) [inline api/admin/admin.ts]: Maybe add all of these functions into AdminPanel?

File: api/admin/handler.ts [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+  const data = event.body;
+
+  console.log('data:', data);
+
+  const adminPanel = new AdminPanel();
+
+
+  adminPanel.createOrder(id, data)
+    .then((data) => callback(null))
```
Comment (azhivaeva-bmt) [inline api/admin/handler.ts]: Maybe return some message, for example, { message: 'Order successfully added' } or return new order

File: serverless.yml [hunk 6]
```diff
@@ -280,5 +311,23 @@ resources:
...
+              - AttributeName: "id"
+                AttributeType: "S"
+            KeySchema:
+              - AttributeName: "id"
+                KeyType: "HASH"
+            ProvisionedThroughput:
+              ReadCapacityUnits: 1
+              WriteCapacityUnits: 1
+            StreamSpecification:
+              StreamViewType: "NEW_AND_OLD_IMAGES"
```
Comment (azhivaeva-bmt) [inline serverless.yml]: You can not use " "

**Pull request merged**